packages/shared/components/popups/MintNftFormPopup.svelte

```diff
@@ -94,11 +98,12 @@
             return
         }
 
-        const isKeysValid = Object.keys(royalties).every((key) => !validateBech32Address($networkHrp, key))
-        if (!isKeysValid) {
+        try {
+            Object.keys(royalties).forEach((key) => validateBech32Address($networkHrp, key))
+        } catch (err) {
             optionalInputErrors.royalties = `Invalid address, must be a valid ${$networkHrp} address where royalties will be sent to.`
-            return
```

Comment:
also remove the other `return`s so we get all errors in one go

Reply:
I actually added a `return` since otherwise it would overwrite the `optionalInputErrors.royalties` string each time. Ideally we should refactor this, so that it throws an error as well, but let's wait until the duplication w/ parseNftMetadata has been removed.